src/lib/translators/base.ts

```diff
@@ -60,6 +60,11 @@ function unsupported(target: BaseTranslator, propertyKey: S.PipelineStepName, de
  * - a `supports` method that tests the support of a given step name.
  */
 export class BaseTranslator implements StepMatcher<OutputStep> {
+  /**
+   * `label` will be display to the suer
+   */
+  get label(): string { return '' }
```

Comment:
I think we should use a `static` class member for labels.

Reply:
I agree! Done ✅